tools/msp_sr_readmap_and_size_histograms/readmap.xml

```diff
@@ -4,6 +4,7 @@
         <requirement type="package" version="1.0.0">bowtie</requirement>
         <requirement type="package" version="0.9.0">pysam</requirement>
         <requirement type="package" version="1.9.3">numpy</requirement>
+        <requirement type="package" version="3.1.2">R</requirement>
```

Comment:
No such package exists in in conda, and conda doesn't need recursive package specification.
r-optparse will pull in R.
The problem you're seeing must be something else.